src/nvidia-cuda/README.md

```diff
@@ -16,7 +16,10 @@ Installs shared libraries for NVIDIA CUDA.
 | Options Id | Description | Type | Default Value |
 |-----|-----|-----|-----|
 | installCudnn | Additionally install CUDA Deep Neural Network (cuDNN) shared library | boolean | false |
+| installCudnnDev | Additionally install CUDA Deep Neural Network (cuDNN) development libraries and headers | boolean | false |
 | installNvtx | Additionally install NVIDIA Tools Extension (NVTX) | boolean | false |
+| installNvcc | Additionally install NVIDIA CUDA Compiler (NVCC) | boolean | false |
```

Comment:
I don't see `installNvcc` added to this PR, wondering if this is intentional or if we are missing some code.

Reply:
This is my mistake. I just remembered that I need to remove this line. CUDA toolkit contains nvcc. So we don't have to install nvcc explicitly.